limbo/config.py

```diff
@@ -0,0 +1,50 @@
+import os
+from backports import configparser as confparser
+
+
+class Config:
+    env_config = {}
+    file_config = None  # type: confparser.ConfigParser
+
+    def __init__(self):
+        self.load_env_config()
+        self.load_config_file()
+
+    def load_config_file(self):
+        self.file_config = confparser.ConfigParser()
+        config_file = self.env_config.get("config_location", "config.ini")
+        # check if config exists
+        import os
```

Comment:
import this at the top of the file, not in here

Reply:
Ok